README.md

```diff
@@ -35,8 +35,8 @@ For more information about spotless checks see
 [here](https://github.com/diffplug/spotless/tree/master/plugin-gradle#custom-rules).
 
 #### Tests
-* Tests are run automatically with `./gradlew build`.
-* To skip all tests, add `-x test`.
+* All tests are run automatically with `./gradlew build`.
+* To skip all tests, add `-x test` and `-x integrationTest`.
```

Comment:
✏️ Does `-x test` skip unit tests and `-x integrationTest` skip itests? I recommend including the command for skipping all tests and the command for just skipping itests.

Reply:
Ah yeah. I clarified that and must have remove it in the readme. I think i can just have test depend on integrationTest so we get:
all tests: ./gradlew build
no itests: ./gradlew build -x test